Implementations/CloudFabric.EventSourcing.EventStore.Postgresql/PostgresqlEventStore.cs

```diff
@@ -40,7 +42,7 @@ public async Task<bool> HardDeleteAsync(Guid streamId, string partitionKey, Canc
         await using var transaction = await conn.BeginTransactionAsync(IsolationLevel.ReadCommitted, cancellationToken);
 
         await using var cmd = new NpgsqlCommand(
-            $"DELETE FROM \"{_tableName}\"" +
+            $"DELETE FROM \"{_eventTableName}\"" +
```

Comment:
In cosmosDb you have eventSContainer (plural), let's make it similar - _eventsTableName